azerothcore/azerothcore-wotlk
# fix(Core/Spells): Fixed applying `Command` aura to shaman Feral Spirits.

Fixes #12851

  

## Issues Addressed:

- Closes #12851
- Closes https://github.com/chromiecraft/chromiecraft/issues/3981

## Tests Performed:

- Tested ingame.

## How to Test the Changes:

Be orc shaman on 80 lvl
`.unaura all`
Use this script to see Spirit Wolf pet tab
`/run if not oldHasPetUI then oldHasPetUI = HasPetUI; HasPetUI = function() return true, false; end end PetTab_Update() ToggleCharacter("PetPaperDollFrame")`
Summon Feral Spirit `.cast 51533`
Check Pet tab for pet's damage
Dismiss pet
`.aura 65222`
`.cooldown`
Summon Feral Spirit `.cast 51533`
see if Damage numbers of pet  have changed


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
